Jenkinsfile

```diff
@@ -362,9 +359,9 @@ pipeline {
                         checkoutScm withSubmodules: true
                         catchError(stageResult: 'UNSTABLE', buildResult: 'SUCCESS') {
                             sh label: env.STAGE_NAME,
-                               script: '''rm -rf artifacts/centos7/
-                                          mkdir -p artifacts/centos7/
-                                          make CHROOT_NAME="epel-7-x86_64" -C utils/rpms chrootbuild'''
+                               script: 'artdir=centos7 ' +
+                                       'chroot_name=epel-7-x86_64 ' +
+                                       'ci/rpm_build.sh'
```

Comment:
Can we move the logic for selecting what chroot_name to use into the rpm_build.sh ?
And then just pass it an environment variable like "TARGET" to indicate what distro / version we are targeting?

Reply:
Certainly seems reasonable.